WordPress/Classes/ViewRelated/Blog/SiteManagement/TableViewHeaderDetailView.swift

```diff
@@ -112,35 +92,21 @@ open class TableViewHeaderDetailView: UITableViewHeaderFooterView {
     }
 
     fileprivate func stackSubviews() {
+
         contentView.addSubview(stackView)
         stackView.addArrangedSubview(titleLabel)
         stackView.addArrangedSubview(detailLabel)
 
-        stackView.centerXAnchor.constraint(equalTo: contentView.centerXAnchor).isActive = true
-        stackWidthConstraint = stackView.widthAnchor.constraint(equalToConstant: UIScreen.main.bounds.width)
-        stackWidthConstraint?.isActive = true
+        NSLayoutConstraint.activate([
+            stackView.leadingAnchor.constraint(equalTo: contentView.readableContentGuide.leadingAnchor),
+            stackView.trailingAnchor.constraint(equalTo: contentView.readableContentGuide.trailingAnchor),
+            stackView.topAnchor.constraint(equalTo: contentView.readableContentGuide.topAnchor),
```

Comment:
Should the `top` and `bottom` anchors be using the `readableContentGuide` here, or just the `contentView` itself? Or does it not matter?